test/passing/tests/wrap_comments.ml.ref

```diff
@@ -33,7 +33,8 @@ let _ =
 [@@@ocamlformat "wrap-comments=false"]
 
 type t =
-  | Aaaaaaaaaa (* Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. *)
+  | Aaaaaaaaaa
+  (* Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. *)
```

Comment:
It is not clear to me that this is a regression. The comment exceeds the margin, doesn't it. In that case, it is perfectly legit to break before it.

Reply:
I think it's because it's now considered to be attached before the next constructor, instead of after the previous one. I've been working on another branch to add a location to the `Parsetree.case` type so that the bar symbol would be part of the location and we wouldn't have dedicated shenanigans in Cmts.ml to handle that, but there were still some regressions.

Reply:
regression fixed